gnome_search_provider/README.md

```diff
@@ -0,0 +1,13 @@
+# Ripasso Gnome Search Provider
+
+WIP search provider for GNOME using ripasso.
+
```

Comment:
I think we need some more instructions on how to use this.

What should a user do after they run the install.sh script, should they enable the systemd services, or is it a gnome setting somewhere to enable it?

A screenshot on how it looks to use the searchbar and get results from ripasso would maybe also be nice?

Reply:
I updated the Readme with instructions and a screenshot. 

Also thank you for making me realize I was working without Clippy enabled in my editor this entire time...

Ideally, the steps in the install.sh script would be baked into distribution packages. For example, I have a PKGBUILD for Arch that does the installation. Much better because they are cleaned up when the package is uninstalled.